### ISSUE TITLE
the enter key on /sign-up page does not  submit the forms.

### ISSUE THREAD

**Issue by reynan-dev** [reynan-dev/codenaute#78]
## Behavior

When typing the sign-up form the use of the enter key only shows the password in password fields, or does nothing in the other fields instead of submit the form as the default.

## Desired

The enter key when used when filling out the form should submit the form.

## How to reproduce

To reproduce this error, you must go to the /sign-up page, fill in all the required fields, and try to submit by pressing the enter key.

**Comment by reynan-dev** [comment]
The same on /sign-in page

**Comment by jkergal** [comment]
Yup, i agree, i think that i already created ticket to improve sign-up / sign-in forms accessibility, but we had bigger priorities :)
